tests/conftest.py

```diff
@@ -3526,10 +3589,10 @@ def bobby_columnar_table_multi_batch(empty_data_context):
                 "meta": {
                     "details": {
                         "success_ratio": 1.0,
-                        "candidate_strings": [
-                            "%Y-%m-%d %H:%M:%S",
-                            "%y-%m-%d",
-                        ],
+                        "candidate_strings": {
+                            "%Y-%m-%d %H:%M:%S": 1.0,
+                            "%y-%m-%d": 0.0,
+                        },
```

Comment:
`datetime` Parameter builder now contains `candidate_strings` as a sorted dictionary of `datetime` strings.